### ISSUE TITLE
Feature Request: Right Click Support For Program

### ISSUE THREAD

**Issue by johnbie** [tjackenpacken/taskbar-groups#103]
Not sure if this is possible, but it would be nice if you can right click the programs icon in the group the same way you can for icons in the taskbar.

Also, is there any way to hide the open programs in the taskbar and only make it show up in the group?

Not going to push two requests on you hard because it seems to be incompatible with the implementation.

**Comment by PikeNote** [comment]
Hiding the program taskbar wise, I don't believe that is possible sadly and can lead to some issues knowing which application is open if it was gone. Taskbar groups was meant to mostly used as a place to group apps and launch it from said places.

With right click context menus (assuming that is what you are talking about), I don't think that is possible but I could be wrong? I'll look into it, but I don't think there is no exact way to do that at the moment.

**Comment by darkmoon2** [comment]
Yeah I am new to the application, and this would be the number one requested feature to me of what is missing in the shortcuts.  I am not sure I know exactly how your accomplishing this tray wizardry yet but it would be super helpful because the biggest use I have for this is keeping all my browser shortcuts so my task bar can contain all my apps, however I need to right click those to go to private so what I am going to try is to add the private links as their own shortcuts.
